### PR TITLE
Add sample: Front Door Standard/Premium (Preview) with Azure Functions origin

### PR DESCRIPTION
**Pull request by johndowns**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Add sample - Front Door Standard/Premium (Preview) with Azure Functions origin

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by bmoore-msft** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by bmoore-msft** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

File: 201-front-door-standard-premium-function-public/readme.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Front Door Standard/Premium (Preview) with Azure Functions origin
```
Comment (bmoore-msft) [inline 201-front-door-standard-premium-function-public/readme.md:1]: change casing on filename

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 201-front-door-standard-premium-function-public/readme.md (hunk 1 above)
Comment (johndowns) [reply 201-front-door-standard-premium-function-public/readme.md]: Done.

**Comment by bmoore-msft** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Pull request merged**